packages/edit-site/src/components/save-button/index.js

```diff
@@ -12,7 +12,7 @@ import { displayShortcut } from '@wordpress/keycodes';
  */
 import { store as editSiteStore } from '../../store';
 
-export default function SaveButton() {
+export default function SaveButton( { showTooltip = true } ) {
```

Comment:
Is there a reason for keeping the tooltip in the header `Save` button when is disabled?

Reply:
yes, there is. In the post editor we want to force the tooltip to show regardless of whether there's a shortcut.